client/web-dev-server.config.mjs

```diff
@@ -38,7 +38,8 @@ export default {
       include: ['src/utils/config.js'],
       preventAssignment: false,
       __api_url__: (process.env.API_URL || 'http://localhost:8000') + '/api',
-      __purchase_mode__: process.env.AVOCANO_PURCHASE_MODE,
+      __purchase_mode__: process.env.AVOCANO_PURCHASE_MODE || 'default',
```

Comment:
As noted above, might want a more descriptive default value here for display in footer.

Reply:
Fixed.